Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml

```diff
@@ -250,6 +250,10 @@
       entity: WallReinforced
       edges:
         - to: girder
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetPlasteel1
+              amount: 4
```

Comment:
seems like this should honestly be an edge from `reinforcedWall` -> `reinforcedGirder` dropping 2 plasteel, then the `reinforcedGirder` -> `girder` edge can drop the other 2 plasteel

Reply:
Yeah that makes more sense, I will do the change as soon as possible, thank you!

Reply:
Actually, there is an issue. The reinforced girder requires a welding step to deconstruct it in a normal girder. That means this change would increase the welding steps from 2 to 3 to completely take down a reinforced wall. Isn't it a bit too much? I think a possible change would be to swap the Welding step with a Cutting step on the reinforced girder -> girder graph, but I don't know if there's anything balance-wise to discuss regarding this.